pkgs/top-level/all-packages.nix

```
@@ -15680,7 +15680,7 @@ in
 
     ### LIBRARIES
 
-    accounts-qt = callPackage ../development/libraries/accounts-qt { };
+    accounts-qt = libsForQt5.callPackage ../development/libraries/accounts-qt { };
```

This needs `callPackage` because it is in `mkLibsForQt5`:

```suggestion
    accounts-qt = callPackage ../development/libraries/accounts-qt { };
```

---

Applied in 73779ab8d4bf97c775d84d9511944c43fd053e1a